### PR TITLE
Bugfix/multiple word auto

### PR DESCRIPTION
**Pull request by paganotoni**
Solves #1000

### CODE REVIEW

File: render/auto.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func (htmlAutoRenderer) ContentType() string {
 }
 
 func (ir htmlAutoRenderer) Render(w io.Writer, data Data) error {
-	name := inflect.Name(strings.ToLower(ir.typeName()))
+	name := inflect.Name(inflect.Underscore(ir.typeName()))
```
Comment (markbates) [inline render/auto.go]: Doesn’t inflect.Name.File() do what we want?

**Comment by paganotoni** [comment]
@markbates when you have the chance I just fixed my test and is passing now

**Comment by markbates** [comment]
@paganotoni looks good. feel free and merge once the tests all pass. thanks.

**Comment by paganotoni** [comment]
thanks @markbates! so good to have issues assigned! do you have something else you want me to look into for v0.11.1 ?

**Pull request merged**